packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -10,6 +10,7 @@
         background-attachment: fixed;
         background-size: cover;
         content: '';
+        height: 100%;
```

Comment:
discussed, this was apparently accidentally removed in a previous PR and is being added back in